.evergreen/evergreen.yml.in

```diff
@@ -1365,6 +1391,36 @@ buildvariants:
       <% } } } %>
       - name: sign_artifact_win32_x64
       - name: sign_artifact_win32msi_x64
+  - name: verify_debian_artifact
+    display_name: "Ubuntu 20.04 x64 (Verification)"
+    run_on: ubuntu2004-small
+    tasks:
```

Comment:
```suggestion
```

alternative suggestion: just make this part of `linux_package`, having it grouped with the packaging and signing tasks is actually pretty reasonable imo

Reply:
addressed feedback in [`eea7fde` (#1961)](https://github.com/mongodb-js/mongosh/pull/1961/commits/eea7fde4c7d583796a4edf0f2dff874073889117)

And i just noticed **just make this** in your comment.